### DISCUSSION TITLE
How to install a local npm module?

### DISCUSSION THREAD

**Discussion by f3k-freek** [quasarframework/quasar#14203] [General - Components / Directives / etc] [answered]
I've got a typescript library/module. If I install it like this it works:
```sh
npm install git+git.example.com:company/psv-game-engine
```

But if I try to install it locally like this:
```sh
npm install ../psv-game-engine
```
I get the following error:
<img width="896" alt="Screenshot 2022-08-16 at 13 19 33" src="https://user-images.githubusercontent.com/61347547/184869121-f129ea13-c9b6-4aa9-8eec-bd929d8d2471.png">

I've also tried to do the "link" route:
```sh
npm link # In module
```
```sh
npm link psv-game-engine # In main project
```
But I get exactly the same error.

So how do I install local modules?

**Comment by MilosPaunovic** [comment, marked as answer]
As this is not at all Quasar related, Google is your friend:
https://stackoverflow.com/questions/53716244/how-to-create-a-local-module-in-typescript

**Reply by f3k-freek** [reply]
Thank you for your fast reply. The link didn't provide a solution as I had already done those steps.
In the end I added/changed the following to tsconfig.json in my module:
```json
{
  "compilerOptions": {
     "module": "es6", 
     "moduleResolution": "Node",
     ...
   },
}
```
